source/llassetgen/include/llassetgen/FontFinder.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <ft2build.h>
+#include FT_FREETYPE_H
+#include <string>
+#ifdef _WIN32
+#include <vector>
+#endif
+
+#include <llassetgen/llassetgen.h>
+#include <set>
```

Comment:
can you actually include FontFinder in your application (cmd) withouth the following line?
#include <llassetgen/llassetgen_api.h>

Reply:
Yeah, works on my machine. If the line is needed by Windows or the GUI, feel free to add it

Reply:
Hey, hab jetzt mal mit Verstand hingeschaut. Also *theoretisch* dürfte dir das um die Ohren fliegen, wenn du die libs fertig baust und dann von außen drauf zugreifst, weil die Klasse nicht in die API exportiert wird, wenn dus weglässt. Dass es trotzdem funktioniert, ist vielleicht schön und gut, aber ich würde die Zeile da hinmachen. (mach ich jetzt einfach mal)